tests/aframe/index.html

```diff
@@ -11,46 +11,46 @@
             <a-entity 
             visible
             id="banner1"
-            zesty-banner="space: 0; creator: 0x0000000000000000000000000000000000000000; format: tall; style: standard; height: 2;" 
+            zesty-banner="space: 999; creator: 0x0000000000000000000000000000000000000000; format: tall; style: standard; height: 2; beacon: true" 
```

Comment:
I have a feeling we might hit space 999 at some point - should we go for a higher number?

Reply:
I don't expect we're going to hit it anytime soon, but if you want to bump it up now we could go for like 9999

Reply:
Go for like 999999